### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by andreszuluaga-wolox**
Add updateCurrentUser action, to update current user profile
Add getProfileUser action, to get data from other user by username

Simulator Screen Shot - iPhone 8 - 2021-09-03 at 15 42 59
Simulator Screen Shot - iPhone 8 - 2021-09-03 at 15 43 11

### CODE REVIEW

File: src/redux/auth/actions.ts [hunk 1]
```diff
@@ -71,5 +75,32 @@ export const actionCreators = {
...
+      })
+    ]
+  }),
+  getUserProfile: (user: UserResponse, successFunction: Function, updateFailed: Function) => ({
+    type: actions.GET_USER_PROFILE,
+    target: TARGETS.USER_PROFILE,
+    payload: user.user?.username,
+    service: getUserProfile,
+    injections: [
+      withPostSuccess((dispatch: Dispatch<any>, response: ApiOkResponse<ProfileData>) => {
```
Comment (john07289) [inline src/redux/auth/actions.ts:96]: replace dispatch for _

File: src/app/screens/Profile/index.tsx [hunk 2]
```diff
@@ -29,23 +32,40 @@ function Profile() {
...
   } = useForm<ProfileFormValues>({ mode: 'onBlur' });
 
+  const usernameExist = () => setUsernameError(i18next.t('PROFILE:USERNAME_EXIST'));
+
+  const emailExist = () => setEmailError(i18next.t('PROFILE:EMAIL_EXIST'));
   const hasError = !!error;
   const handleUpdateProfile = (values: ProfileFormValues) => {
     Keyboard.dismiss();
-    updateProfile(values);
+    // updateProfile(values);
```
Comment (EstebanIndiveri) [inline src/app/screens/Profile/index.tsx:52]: delete this

**Review by EstebanIndiveri — APPROVED** [review]
good job

File: src/app/screens/Profile/index.tsx (hunk 2 above)
Comment (andreszuluaga-wolox) [reply src/app/screens/Profile/index.tsx]: done!

**Review by john07289 — APPROVED** [review]

**Pull request merged**